src/WorldStorage/NamespaceSerializer.cpp

```diff
@@ -501,7 +501,7 @@ eMonsterType NamespaceSerializer::ToMonsterType(const std::string_view a_ID)
 		{ "zoglin",           mtZoglin },
 		{ "zombie",           mtZombie },
 		{ "zombie_horse",     mtZombieHorse },
-		{ "zombie_pigman",    mtZombiePigman },
+		{ "zombified_piglin", mtZombiePigman },
```

Comment:
this was intended to provide support for old save files. the other version is further up/down

Reply:
I see a "PigZombie" mapping. Were we saving as "zombie_pigman" before?

Reply:
Just in case someone tried to load a map from a version that uses the name

Reply:
So I'd suggest keeping both

Reply:
Done